pkg/qemu/qemu.go

```diff
@@ -208,12 +210,24 @@ func Cmdline(cfg Config) (string, []string, error) {
 	}
 	switch *y.Arch {
 	case limayaml.X8664:
-		cpu := "Haswell-v4"
+		cpu := "qemu64"
 		if isNativeArch(*y.Arch) {
 			cpu = "host"
 		}
 		args = appendArgsIfNoConflict(args, "-cpu", cpu)
-		args = appendArgsIfNoConflict(args, "-machine", "q35,accel="+accel)
+		if isNativeArch(*y.Arch) {
+			args = appendArgsIfNoConflict(args, "-machine", "q35,accel="+accel)
+		} else {
+			// use q35 machine with vmware io port disabled.
+			args = appendArgsIfNoConflict(args, "-machine", "q35,vmport=off")
+			// use tcg accelerator with multi threading with 512MB translation block size
+			// https://qemu-project.gitlab.io/qemu/devel/multi-thread-tcg.html?highlight=tcg
+			// https://qemu-project.gitlab.io/qemu/system/invocation.html?highlight=tcg%20opts
+			// this will make sure each vCPU will be backed by 1 host user thread.
+			args = appendArgsIfNoConflict(args, "-accel", "tcg,thread=multi,tb-size=512")
+			// This will disable CPU S3 state.
```

Comment:
For what purpose?

Reply:
When running emulations I found that the VM doesn't use more than 125%.  Which seems that nor more than 1 thread is used from host vm for emulation. But adding this tcg accel cause the VM to use more host threads (CPUs). After this change when running the VM load i found the CPU usage spike up to 300% - 400% on the same apple silicon machine. I found this when comparing the load test on the VM created using UTM App and Lima. The last s3 state toggle might not effect the performance directly. But may be it will help the VM to run without going into s3 state.